### PR TITLE
Bugfix/embedded

### PR DESCRIPTION
**Pull request by maluramichael**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| License       | MIT

### CODE REVIEW

File: Api/IriConverter.php [hunk 1]
```diff
@@ -11,6 +11,7 @@
 
 namespace Dunglas\ApiBundle\Api;
 
+use AppBundle\Entity\GeoCircle;
```
Comment (teohhanhui) [inline Api/IriConverter.php]: Please remove.

File: Api/ResourceResolverTrait.php [hunk 2]
```diff
@@ -43,6 +43,7 @@
      */
     public function guessResource($object, array $context = null, $strict = false)
     {
+
```
Comment (teohhanhui) [inline Api/ResourceResolverTrait.php]: Unwanted blank line.

File: Api/IriConverter.php [hunk 3]
```diff
@@ -98,16 +99,22 @@ public function getItemFromIri($iri, $fetchData = false)
     public function getIriFromItem($item, $referenceType = RouterInterface::ABSOLUTE_PATH)
     {
         if ($resource = $this->resourceCollection->getResourceForEntity($item)) {
+
```
Comment (teohhanhui) [inline Api/IriConverter.php]: Unwanted blank line.

**Comment by teohhanhui** [comment]
It's unclear what you're trying to achieve, but you've introduced BC breaks, which is not acceptable in 1.x

File: Api/IriConverter.php [hunk 4]
```diff
@@ -98,16 +99,22 @@ public function getItemFromIri($iri, $fetchData = false)
...
             $identifier = $this->getIdentifierFromResource($resource);
 
+            if(!$identifier)
```
Comment (soyuka) [inline Api/IriConverter.php]: code style, `if (`, same below

**Comment by teohhanhui** [comment]
@api-platform/core-team OK to close this? The author doesn't seem to be responsive, and there's no clear description of what he's trying to achieve.

**Comment by dunglas** [comment]
ok for me

**Pull request closed without merging**